server/src/server/deduplicator/AttributeDeduplicatorDaemon.java

```diff
@@ -80,16 +80,16 @@ public AttributeDeduplicatorDaemon(SessionFactory sessionFactory) {
      * @param index the value of the attribute
      * @param conceptId the concept id of the attribute
      */
-    public void markForDeduplication(KeyspaceImpl keyspace, String index, ConceptId conceptId) {
-        Attribute attribute = Attribute.create(keyspace, index, conceptId);
+    public void markForDeduplication(KeyspaceImpl keyspace, String label, String index, ConceptId conceptId) {
+        Attribute attribute = Attribute.create(keyspace, label, index, conceptId);
         LOG.trace("insert({})",  attribute);
         queue.insert(attribute);
     }
 
     /**
      * Starts a daemon which performs deduplication on incoming attributes in real-time.
      * The thread listens to the RocksDbQueue queue for incoming attributes and applies
```

Comment:
I know this is not your change, but can you rename the rocksdbqueue to just queue - I forgot to change this